app/views/request/_sidebar.html.erb

```diff
@@ -1,49 +1,66 @@
 <div id="right_column" class="sidebar right_column" role="complementary">
-  <div class="new-request-cta">
+  <div class="sidebar__section new-request-cta">
     <%= render :partial => "general/new_request" %>
   </div>
 
+  <% if show_pro_upsell?(@user) %>
+    <div class="sidebar__section pro-upsell">
+      <% pro_link = link_to account_request_index_path do %>
+        <%= AlaveteliConfiguration.pro_site_name -%>
+      <% end %>
+
+      <% pro_site_name_link = content_tag(:strong, pro_link) %>
+
+      <%= _('{{pro_site_name_link}} is a powerful, fully-featured FOI ' \
```

Comment:
yay, this exact wording is used on the account request page so we can go ahead and lock the translations down for the release when we're ready